### DISCUSSION TITLE
Is it advisable to use KeychainSettings as a general purpose persistent storage?

### DISCUSSION THREAD

**Discussion by BitByAVampire** [russhwolf/multiplatform-settings#215] [Q&A]
KeychainSettings implement the Settings interface, but underneath it uses the Keychain API, which is intended for encrypting sensitive information. However, theres a lot of information that is not as sensitive as passwords. should they use the keychainsettings?

**Comment by russhwolf** [comment]
The library tries to be unopinionated here. The NSUserDefaults implementation is there for when you don't want encryption, and the Keychain implementation is there for when you do. But it's up to you to determine which you need for your specific use-case. As a library maintainer I don't want to be in the business of making security recommendations, only providing options.
